.golangci.yml

```diff
@@ -0,0 +1,282 @@
+# options for analysis running
+run:
+  # default concurrency is a available CPU number
+  # concurrency: 4
```

Comment:
Should we enable this? Don't suppose there's some way to take this from an env var so that we can run CI with high concurrency?

Reply:
you can override this on the command line for CI